src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -829,6 +829,10 @@ private int RunCore(TextWriter consoleOutput, ErrorLogger errorLogger, Cancellat
                     return Failed;
                 }
             }
+            catch (IOException e)
```

Comment:
It looks like PeWriter.WritePeToStream already guards one call to WriteContentTo, but there is another unguarded call. Perhaps we should expand the guard?

---
In reply to: [120700402](https://github.com/dotnet/roslyn/pull/20075#discussion_r120700402) [](ancestors = 120700402)